es-design-system/pages/organisms/es-form.vue

```diff
@@ -258,6 +318,21 @@ export default {
                 }
             }, 5000);
         },
+        getErrorMessage(validatorName) {
+            const ERROR_MESSAGES = {
+                [vuelidateKeys.REQUIRED]: () => 'This field is required',
+                [vuelidateKeys.MIN_LENGTH]: (min = 8) => `This field must be at least ${min} characters`,
+                [vuelidateKeys.HAS_NUMBER]: () => 'This field must include a number',
+                [vuelidateKeys.HAS_SPECIAL_CHARACTER]: () => 'This field must include a special character',
+                [vuelidateKeys.HAS_UPPERCASE_LETTER]: () => 'This field must include an uppercase letter',
+                [vuelidateKeys.HAS_LOWERCASE_LETTER]: () => 'This field must include a lowercase letter',
+            };
+            const msgFunc = ERROR_MESSAGES[validatorName];
+            if (msgFunc) {
+                return msgFunc();
+            }
+            return '';
+        },
```

Comment:
This feels like it could be part of a mixin.

Reply:
Yeah, I haven't thought through how this would interface with i18n either. I can pull it out.